securedrop_client/gui/__init__.py

```diff
@@ -176,10 +176,10 @@ def __init__(
         self.setText(text)
         self.elided = True if self.text() != text else False
 
-    def setText(self, text: str) -> None:
+    def setText(self, text: str, wrap_limit: int = 80) -> None:
```

Comment:
since the constructor calls `setText` i think it would make more sense to add this parameter to the constructor so that if we create a QSecureLabel liek this: `QSecureLabel('hello i am text that needs to be wrapped etc etc')` then it will use the wrap_limit provided